.github/workflows/coverage.yml

```diff
@@ -17,8 +17,8 @@ jobs:
         with:
           coverage: pcov
       - name: Install dependencies
-        run: composer install
-      - name: "Run test suite"
```

Comment:
BTW I quote all YAML strings. https://github.com/szepeviktor/byte-level-care/blob/master/.github/workflows/back-end.yml

Reply:
Can you explain why this would have an advantage over not quoting YAML strings?

Reply:
My name is Explicit Viktor.
Explicit is better than implicit.
Nominal format is always safer and causes less mental load: don't have to to decide whether quote or not.